frontend/src/components/VideoStream/VideoSection.tsx

```diff
@@ -23,15 +25,37 @@ const FullscreenButton = styled(Button)`
     opacity: 0.75;
 `
 
-Icon.add({ fullscreen })
+const NoVideoPlaceholder = styled(Icon)`
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    width: 50%;
+    height: 50%;
+    opacity: 0.4;
+`
 
 export function VideoSection({ videoStream, toggleFullScreenModeFunction }: VideoSectionProps) {
     return (
         <StyledVideoSection>
-            <img src={videoStream.url} width="100%" height="100%" alt={videoStream.name + ' video stream'} />
-            <FullscreenButton color="secondary" onClick={toggleFullScreenModeFunction}>
-                <Icon name="fullscreen" size={32} />
-            </FullscreenButton>
+            {ValidateVideoStream(videoStream) && (
+                    <img src={videoStream.url} width="100%" height="100%" alt={videoStream.name + ' video stream'} />
+                ) && (
+                    <FullscreenButton color="secondary" onClick={toggleFullScreenModeFunction}>
+                        <Icon name="fullscreen" size={32} />
+                    </FullscreenButton>
+                )}
+            {!ValidateVideoStream(videoStream) && <NoVideoPlaceholder name="videocam_off" />}
         </StyledVideoSection>
     )
 }
+
+function ValidateVideoStream(videoStream: VideoStream) {
+    const stream = new Image()
+    stream.src = videoStream.url
+    if (stream.naturalHeight === 0) {
```

Comment:
Is this a safe check for validating video stream?

Reply:
Yes, if the image is available then it must return naturalHeight > 0.
However, I don't have a video stream stack set up on my setup, so I won't be able to check for edge cases.
If we find some, we can always augment the validate function was my thinking.

Reply:
Okidoki